Documentation/GeneralConventions/FileStructure.rst

```diff
@@ -651,31 +651,13 @@ the toolchain log file at :file:`Documentation-GENERATED-temp/Result/project/0.0
 .. index:: Single file documentation
 .. _single-file-documentation:
 
-Single file documentation
-=========================
+Single file documentation (README)
+==================================
 
-This setup is not recommended, but can be handy for those who want to publish
-their documentation on docs.typo3.org and want to keep their mono
-:file:`README.rst` documentation style for now.
+For third-party TYPO3 extensions that do not require extended documentation
+you can also publish a :file:`README.rst` or :file:`README.md` to
```

Comment:
Maybe add that these can be in the extension's root folder. The Documentation folder can then be omitted. But you can also use it.